recipe-server/client/control/components/recipes/RecipeDetails.js

```diff
@@ -30,6 +30,50 @@ export default class RecipeDetails extends React.PureComponent {
     }
   }
 
+  renderArgumentValue(name, value) {
+    switch (name) {
+      case 'branches': {
+        const sumRatios = value.map(branch => branch.get('ratio')).reduce((a, b) => a + b) || 1;
+        return (
+          <table className="pref-experiment-branches">
```

Comment:
I'm pretty happy with how this turned out:

<img alt="screenshot-2017-10-24 shield control panel 1" src="https://user-images.githubusercontent.com/305049/31970661-939016c4-b8cd-11e7-975b-ba3912a0b31a.png" width="50%">